test-e2e/members.js

```diff
@@ -155,3 +155,50 @@ test('getting invited member after invite accepted', async (t) => {
   // TODO: Test that device info of invited member can be read from invitor after syncing
   await disconnectPeers(managers)
 })
+
+test('invite with abritrary name', async (t) => {
+  t.plan(1)
+  const managers = await createManagers(2, t)
+  const [invitor, invitee] = managers
+  connectPeers(managers)
+  await waitForPeers(managers)
+
+  const projectId = await invitor.createProject({ name: 'Mapeo' })
+
+  const inviteResponse = await invite({
+    invitor,
+    projectId,
+    invitees: [invitee],
+    roleName: 'friend',
+    reject: true,
+  })
+
+  t.is(inviteResponse?.roleName, 'friend', 'roleName should be equal')
```

Comment:
would avoid updating the `invite()` utility and do this instead:

```suggestion
  invitee.invite.on('invite-received', ({ roleName }) => {
    t.is(roleName, 'friend', 'roleName should be equal')
  })
  
  await invite({
    invitor,
    projectId,
    invitees: [invitee],
    roleName: 'friend',
    reject: true,
  })
```